DataConnectors/CiscoISE/Connector_Cisco_ISE.json

```diff
@@ -0,0 +1,113 @@
+{
+    "id": "CiscoISE",
+    "title": " Cisco Identity Services Engine",
+    "publisher": "Cisco",
+    "descriptionMarkdown": "The Cisco Identity Services Engine (ISE) data connector provides the capability to ingest [Cisco ISE](https://www.cisco.com/c/en/us/products/security/identity-services-engine/index.html) events with Azure Sentinel. It helps you gain visibility into what is happening in your network, such as who is connected, which applications are installed and running, and much more. Refer to [Cisco ISE logging mechanism documentation](https://www.cisco.com/c/en/us/td/docs/security/ise/2-7/admin_guide/b_ise_27_admin_guide/b_ISE_admin_27_maintain_monitor.html#reference_BAFBA5FA046A45938810A5DF04C00591) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **CiscoISEEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/CiscoISE/CiscoISE.txt)  ",
```

Comment:
Please use "**https://aka.ms/sentinel-ciscoise-parser**" link for CiscoISEEvent Kusto Function.

Reply:
done